smart-connector/src/test/java/eu/knowledge/engine/smartconnector/runtime/messaging/MockSmartConnector.java

```diff
@@ -189,15 +189,15 @@ public void setDomainKnowledge(Set<Rule> someDomainKnowledge) {
 	}
 
 	@Override
-	public void setReasonerEnabled(boolean aReasonerEnabled) {
+	public void setReasonerLevel(int aReasonerLevel) {
 		// TODO Auto-generated method stub
 
 	}
 
 	@Override
-	public boolean isReasonerEnabled() {
+	public int getReasonerLevel() {
 		// TODO Auto-generated method stub
-		return false;
+		return 0;
```

Comment:
Perhaps we should choose to return the default reasoner level here (2) instead of 0 which is an invalid level?